lib/new_relic/harvest/telemetry_sdk/dimensional_metrics/harvester.ex

```diff
@@ -0,0 +1,87 @@
+defmodule NewRelic.Harvest.TelemetrySdk.DimensionalMetrics.Harvester do
+  use GenServer
+
+  @moduledoc false
+
+  alias NewRelic.Harvest
+  alias NewRelic.Harvest.TelemetrySdk
+
+  @interval_ms 5_000
```

Comment:
I set this to match the harvest cycle time. It's required for `count` and `summary` types in the `common` field.